installer/conf/omsagent.d/npmd.conf

```diff
@@ -0,0 +1,14 @@
+<source>
+    type npmd
```

Comment:
Your latest commit still does not have the change for this plugin and the conf file be deployed during the omsagent install. You need to add them to base.data.